### ISSUE TITLE
Create official docker containers

### ISSUE THREAD

**Issue by steve-gray** [node-freight/freight-server#25]
Hey,

I've created a docker container image for freight that's a CentOS+Redis+freight-server starting point. Are there any intentions to create an official docker container image so I can park/retire the homebrew?

https://github.com/bringthedip/docker-freightserver

**Comment by vladikoff** [comment]
Woa coool!

**Comment by vladikoff** [comment]
Yeah this would be awesome to have those officially available, I'll try to use your docker image and go from there

**Comment by steve-gray** [comment]
Feel free to fork my repo under your node-freight organisation and use that as your starting point. You'll need to get Docker Hub accounts sorted to do a published version etc though.
